### PR TITLE
Add saved filters modal and dashboard widget

### PR DESCRIPTION
**Pull request by kshepard**

### CODE REVIEW

File: web/app/scripts/saved-filters/saved-filters-modal-controller.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    /* ngInject */
+    function SavedFiltersModalController($modalInstance, $rootScope,
+                                         FilterState, Notifications, SavedFilters) {
+        var ctl = this;
+        ctl.label = '';
+        ctl.save = save;
+
+        // linter complains if this function is defined below for some reason
+        ctl.close = function() { $modalInstance.close(); };
```
Comment (ddohler) [inline web/app/scripts/saved-filters/saved-filters-modal-controller.js]: That's weird; if it was defined the same ways as `save` is I don't see why there would be a problem. Does changing the name of the function (maybe `closeModal`) help?

**Comment by ddohler** [comment]
+1, great.

File: web/app/scripts/saved-filters/saved-filters-modal-controller.js (hunk 1 above)
Comment (kshepard) [inline web/app/scripts/saved-filters/saved-filters-modal-controller.js]: Yeah, totally bizarre. I'll try changing the name to see what happens.

Comment (kshepard) [inline web/app/scripts/saved-filters/saved-filters-modal-controller.js]: Works fine with a different name. Guess I'll go with that!

**Pull request merged**